### ISSUE TITLE
Issue in Tearsheet component at 400% zoom

### ISSUE THREAD

**Issue by Vinodbs** [carbon-design-system/ibm-products#4898]
**What package(s) are you using?**

```
@carbon/ibm-products": "^2.27.0",
"react": "^17.0.2",
```
**Detailed description**
When the browser zoom is used to scale content to 400% , the content in the tearsheet gets distorted. The scroll does not work.

> Is this issue related to a specific component?
> Tearsheet

> What did you expect to happen? What happened instead? What would you like to
> see changed?
> When the browser zoom is used to scale content to 400%, the component must reflows - i.e., it should presented in one column so that scrolling in more than one direction is not necessary. The scroll should work.

> What version of the @carbon/ibm-products (or @carbon/ibm-cloud-cognitive)
> package are you using?
> @carbon/ibm-products": "^2.27.0",

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?
> We work for GI team and above issue is raised as a AVT issue

**Comment by sstrubberg** [comment]
Dupe https://github.com/carbon-design-system/ibm-products/issues/3567
